Client/RabbitMqClient.php

```diff
@@ -80,22 +80,27 @@ public function countNotTakenMessages(string $queueName)
     /**
      * @param string $queueName
      * @param AMQPMessage[] $messageList
+     * @param array $contextInfoByTagList
      * @param int $delay
      */
     public function rewindList(
         string $queueName,
         array $messageList,
+        array $contextInfoByTagList,
```

Comment:
- [ ] Поставить на последнее место со значением по умолчанию, чтобы избежать BC
- [ ] Изменить название
```suggestion
        array $deliveryTagContextList,
```